lib/internal/process/warning.js

```diff
@@ -28,11 +29,16 @@ function setupProcessWarnings() {
 
   // process.emitWarning(error)
   // process.emitWarning(str[, name][, ctor])
```

Comment:
You added a new line just below it:
```
// process.emitWarning(str[, name][, ctor][, warned])
```
That covers
```
// process.emitWarning(str[, name][, ctor])
```
so that one was not needed.

The diff now looks like
```diff
  // process.emitWarning(str[, name][, ctor])
+ // process.emitWarning(str[, name][, ctor][, warned])
```
but it should have been

```diff
- // process.emitWarning(str[, name][, ctor])
+ // process.emitWarning(str[, name][, ctor][, warned])
```